pkg/morph/client/container/nodes.go

```diff
@@ -0,0 +1,50 @@
+package container
+
+import (
+	"fmt"
+
+	"github.com/nspcc-dev/neofs-node/pkg/morph/client"
+)
+
+// AddNextEpochNodes registers public keys as a container's placement vector
+// with specified index. Registration must be finished with final
+// [Client.CommitContainerListUpdate] call. Always sends a notary request with
+// Alphabet multi-signature.
```

Comment:
and is always signed by one of the Alphabet node in fact. i added this like because we have `AsAlphabet` and smth like this option for the client. this line highlights that it is always sent by the alphabet node, SN and any other client should not so this